i18n-xgettext-web/src/main/java/net/jhorstmann/i18n/xgettext/web/WebMessageExtractor.java

```diff
@@ -32,9 +32,25 @@
         String ns = DEFAULT_NAMESPACE;
         functions.add(MessageFunction.fromEL(ns, "mark(message)"));
         functions.add(MessageFunction.fromEL(ns, "tr(message)"));
+        functions.add(MessageFunction.fromEL(ns, "tr1(message)"));
```

Comment:
Good catch. Maybe it makes sense to include the additional parameters in the signature here to make it clearer what is being matched. The `fromEL` function only treats parameter names "message", "context" and "plural" specially I think, so the following should still work:

    tr2(message, param1, param2)